recipe/meta.yaml

```diff
@@ -56,6 +56,9 @@ requirements:
     - xattr >=0.10.0,<0.11.0  # [darwin]
     - urllib3 >=1.26.0,<2.0.0
 
+    # `poetry` entry point fails without this:
+    - lockfile >=0.9
```

Comment:
I understand that this is necessary, but I'm confused how Poetry works when this isn't an explicit dependency. Do you understand where it's coming from? Maybe there is a missing dependency from `filelock` or something?

Reply:
I reported this upstream in https://github.com/python-poetry/poetry/issues/7171. Let's see what they say.

I also think it's a problem regardless of whether this should come from another dependency because it's an explicit dependency.

Reply:
Good point, explicit dependencies should definitely be included by the package using them. But it's probably also an implicit dependency which is missing somewhere on CF... I can try to have a look...